### ISSUE TITLE
cell type identification

### ISSUE THREAD

**Issue by hernandezvargash** [yqq-chen/CRC_ICB#1]
Thanks for sharing the code for your article.
Can you please explain how defined the major cell types and subtypes in your annotations?
In particular, how did you distinguish your tumor cells from other epithelial cells?
Thanks!

**Comment by 18parkky** [comment]
@hernandezvargash
**I am not the author of this study**, but I had the same question (and still do to some extent).

In another [single-cell CRC study published in Nat. Genet.](https://doi.org/10.1038/s41588-022-01100-4), the authors note that normal epithelial cells cluster together, regardless of the patient. These cells are mostly derived from the normal adjacent samples, with only a few coming from tumor samples. All other clusters form patient-specifically, which represent the tumor cells from each patient. The authors state that this is not due to batch effects, since all other cell types cluster properly without any batch effects (Extended Data Fig. 2). Instead, this patient specificity reflects intertumoral heterogeneity, not batch effects.

> One cluster comprised all cells from normal samples and a minority of cells from tumor samples (23.4%). These represent normal cells in tumor sectors.

You can find similar logic in other papers as well, and I think this is currently the most solid approach.
If you are unsure, you could also apply inferCNV for MSS CRC, but keep in mind that [most MSI CRC patients and some MSS CRC patients do not exhibit CNVs](https://www.nature.com/articles/s41586-024-07747-9/figures/2).

TL;DR: When you sub-cluster epithelial cells, you'll typically find that one cluster consists of cells from all patients (=normal cells), while all other clusters are patient-specific (=tumor cells). You can also use inferCNV for validation.

I’m currently working on a Python tool to identify MSI CRC cells from single-cell data, so I’ll be exploring similar challenges. Let me know if you’d like to discuss further!

**Comment by hernandezvargash** [comment]
Thank you @18parkky for your comment. Yes, that makes a lot of sense. 
Looking forward to your python tool!
